guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: guide
+title: Creating a new wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+# Creating a new wallet
+
+First, bitcoin [wallets]({{ 'guide/glossary/#wallet' | relative_url }}) are not analogous to their namesakes. Physical wallets are places to store physical currency, credit cards, IDs, and so on. Bitcoin wallets don't store bitcoin.
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin. The bitcoin is located on the Bitcoin network that contains a public ledger similar in many ways to a spreadsheet with a record of everyone's transactions and balances. Private keys let you access bitcoin associated with those keys. A more suitable analogy for a bitcoin wallet would be a keychain.
+
+ Like regular keys on a keychain, bitcoin private keys can be copied. For example, if you gave someone a copy of your house keys, they can unlock the door to your home. Likewise, if someone has a copy of your private key, they can access your bitcoin. This makes [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) one of the most important considerations of any bitcoin owner.
+
+There are a few different types of wallets that let you send, receive, store and manage bitcoin.
```

Comment:
You don't explain what those types are...? Do you mean categories of wallets, or just different apps like Blue, Muun, Core, etc.

Reply:
Good point. In terms of types i'm thinking more...
 - Multi-Sig
 - Watch-only
 - Desktop
 - Paper
 As opposed to specfic vendors, a bit cautious to just list them if we don't places with clear defintions of them. I need to go through the glossary again maybe.

Reply:
A simple solution could be:

`There are a few different types of wallets that let you send, receive, store and manage bitcoin. For this page, we focus on regular single-key wallets which are the ones most commonly used.`